### PR TITLE
Add security marks samples

### PR DESCRIPTION
**Pull request by arthurlapertosa**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ ] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [ ] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: security-center/snippets/system-test/v2/findings.test.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+// Copyright 2019 Google LLC
```
Comment (rafaelMurata) [inline security-center/snippets/system-test/v2/findings.test.js]: using the old version of the header. The new one starts with 'copyright 2024 Google LLC'.

File: security-center/snippets/system-test/v2/findings.test.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+const {SecurityCenterClient} = require('@google-cloud/security-center').v2;
+const {assert} = require('chai');
+const {describe, it, before} = require('mocha');
+const {execSync} = require('child_process');
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
+const organizationId = process.env['GCLOUD_ORGANIZATION'];
+
+describe('Client with SourcesAndFindings', async () => {
```
Comment (rafaelMurata) [inline security-center/snippets/system-test/v2/findings.test.js]: change the describe message to "'client with security marks v2'". And the file name to securityMarks.test.js

File: security-center/snippets/system-test/v2/findings.test.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    const [untouchedFinding] = await client
+      .createFinding(createFindingTemplate)
+      .catch(error => console.error(error));
+    data = {
+      orgId: organizationId,
+      sourceName: source.name,
+      findingName: finding.name,
+      untouchedFindingName: untouchedFinding.name,
+    };
+    console.log('my data %j', data);
```
Comment (rafaelMurata) [inline security-center/snippets/system-test/v2/findings.test.js]: Please add the message "my data security marks" to the log to make it different from other logs.

File: security-center/snippets/system-test/v2/findings.test.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    const eventTime = new Date();
+    const createFindingTemplate = {
+      parent: source.name,
+      findingId: 'somefinding',
+      finding: {
+        state: 'ACTIVE',
+        // Resource the finding is associated with.  This is an
+        // example any resource identifier can be used.
+        resourceName:
+          '//cloudresourcemanager.googleapis.com/organizations/11232',
```
Comment (rafaelMurata) [inline security-center/snippets/system-test/v2/findings.test.js]: please change organization number 11232 to organizationId.

File: security-center/snippets/system-test/v2/findings.test.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+      untouchedFindingName: untouchedFinding.name,
+    };
+    console.log('my data %j', data);
+  });
+
+  it('client can add security marks to finding', () => {
+    const output = exec(
+      `node v2/addFindingSecurityMarks.js ${data.findingName}`
+    );
+    assert.match(output, new RegExp(data.findingName));
```
Comment (rafaelMurata) [inline security-center/snippets/system-test/v2/findings.test.js]: creating a regex from an environment variable throws a security error. Please change it to `assert(output.includes(data.finding Name));
`https://github.com/GoogleCloudPlatform/nodejs-docs-samples/security/code-scanning/83

File: security-center/snippets/system-test/v2/findings.test.js (hunk 1 above)
Comment (arthurlapertosa) [reply security-center/snippets/system-test/v2/findings.test.js]: done

File: security-center/snippets/system-test/v2/findings.test.js (hunk 2 above)
Comment (arthurlapertosa) [reply security-center/snippets/system-test/v2/findings.test.js]: done

File: security-center/snippets/system-test/v2/findings.test.js (hunk 3 above)
Comment (arthurlapertosa) [reply security-center/snippets/system-test/v2/findings.test.js]: done

File: security-center/snippets/system-test/v2/findings.test.js (hunk 4 above)
Comment (arthurlapertosa) [reply security-center/snippets/system-test/v2/findings.test.js]: done

File: security-center/snippets/system-test/v2/findings.test.js (hunk 5 above)
Comment (arthurlapertosa) [reply security-center/snippets/system-test/v2/findings.test.js]: done

**Pull request closed without merging**